src/log.h

```diff
@@ -62,7 +62,7 @@ class StreamProxy {
 		// These calls explicitly use the templated version of operator<<,
 		// so that they won't use the overloads created by ADD_NULL_CHECK.
 		if (arg == nullptr)
-			return this->operator<< <const char*> ("(null)");
+			return this->operator<< <const char*> ((const char*)"(null)");
```

Comment:
Why?

Reply:
Ah, the code won't compile on my end because of this line, so I changed it. It shouldn't be part of this PR though...